### ISSUE TITLE
Empty output

### ISSUE THREAD

**Issue by orl1k** [ultram4rine/sqltools-clickhouse-driver#377]
Hi! All my sql queries return empty output after last update (0.3.0). 
<img width="618" alt="image" src="https://user-images.githubusercontent.com/46086189/231170702-9b3d2f31-27b9-4cea-853f-b5ad1034b9b8.png">

**Comment by polRk** [comment]
The same if i use WITH

**Comment by ultram4rine** [comment]
Hi @orl1k, @polRk, can you provide your queries?
Since in last update I migrated to [@ClickHouse/clickhouse-js](https://github.com/ClickHouse/clickhouse-js), I'm using different methods for different types of queries:
https://github.com/ultram4rine/sqltools-clickhouse-driver/blob/241e527726778b5c259021f6fa83257b9a489b85/src/ls/driver.ts#L59-L103
If `query` starts with `SELECT` then I use `query` [method](https://clickhouse.com/docs/en/integrations/language-clients/nodejs#query-method), which has output, for others I use `exec` [method](https://clickhouse.com/docs/en/integrations/language-clients/nodejs#exec-method), which doesn't have output.
But as I can see it was too reckless.

**Comment by Leonqn** [comment]
So, writing `select` in lowercase also gives no results :)

**Comment by ultram4rine** [comment]
@Leonqn oops, my bad, didn't notice that
I guess that can be fixed easily, so I will patch and release asap.

**Comment by Leonqn** [comment]
Also if you have newlines or spaces in the beginning of a query it also won't work.

**Comment by ultram4rine** [comment]
> Also if you have newlines or spaces in the beginning of a query it also won't work. 

Yeah, also I can't uppercase whole query since that will change tables names, so it is needed to split query and take first item I believe

**Comment by Leonqn** [comment]
I think you can uppercase it like this
```
query.toString().trimStart().toUpperCase().startsWith("SELECT") ? "query" : "exec";
```
It is a local uppercasing and won't change original query

**Comment by ultram4rine** [comment]
@orl1k, @polRk , @Leonqn please check #378.

**Comment by stoffeastrom** [comment]
This does not work with code lens

```sql
-- @block
-- @conn clickhouse
SELECT * FROM X;
```
